README.md

```diff
@@ -96,3 +96,10 @@ to the console)
 
 It is also quite handy to add a shortcut in QtCreator to launch qhot. See
 [Keyboard Shortcuts](https://doc.qt.io/qtcreator/creator-keyboard-shortcuts.html).
+
+### Shortcuts
+
+- **F1** Launch popup to change background
+- **F2** Toggles visibility: maximized, full screen, back to default
+- **F3** After changing window size, reset window to loaded component size
```

Comment:
From the code they are ALT+F1, since ALT+F1 is a common shortcut between window managers, I believe that we should change the code to use only F1, F2 and etc.